src/geometry.rs

```diff
@@ -51,7 +51,29 @@ pub struct ConstGeometry<'a> {
     pub(crate) original: &'a Geometry,
 }
 
-pub trait Geom: AsRaw<RawType = GEOSGeometry> {
+pub trait OriginalGeometry {
+    fn original(&self) -> &Geometry;
+}
+
+impl OriginalGeometry for Geometry {
+    fn original(&self) -> &Geometry {
+        self
+    }
+}
+
+impl OriginalGeometry for ConstGeometry<'_> {
+    fn original(&self) -> &Geometry {
+        self.original
+    }
+}
+
+unsafe impl Send for Geometry {}
```

Comment:
Is it safe to implement `Send`  and `Sync` on these two types?

Reply:
That's not a new introduction, it was already implemented for both 

ttps://github.com/georust/geos/pull/198/files#diff-434e341aaed8371258b2ef24e2c036049c8a56b46d90a2777954ecd6de5afb72L2691-L2692